lib/Locale/Po4a/AsciiDoc.pm

```diff
@@ -481,7 +481,7 @@ sub parse {
             $self->pushline( $line . "\n" );
         } elsif ( ( defined $self->{type} )
             and ( $self->{type} eq "Table" )
-            and ( $line !~ m/^\|===/ )
+            and ( $line !~ m/^\|={3,}$ )
```

Comment:
This part is ok, as discussed. But not the one about source section, which the test is targeting.